vatools/vcf_genotype_annotator.py

```diff
@@ -64,7 +64,7 @@ def main(args_input = sys.argv[1:]):
             if isinstance(entry.FORMAT, tuple):
                 entry.FORMAT = ["GT"]
             else:
-                entry.FORMAT = entry.FORMAT.appened('GT')
+                entry.FORMAT.insert(0, 'GT')
```

Comment:
Could this use vcfpy's [`add_format`](https://vcfpy.readthedocs.io/en/stable/api_record.html#vcfpy.Record.add_format)?

Alternatively: in what circumstance does the format turn out to be a `tuple`?  It surprises me that we'd want to discard the existing value in that case.

Reply:
Good point.

It's a tuple on VCFs without FORMAT or sample fields.

Reply:
on seconds thought, I think I prefer the existing solution because it puts the GT field at the first position (which is where it is usually located) instead of at the end.